examples/building-sass-files/public/manifest.json

```diff
@@ -0,0 +1,10 @@
+{
+  "entrypoints": {
+    "styles": {
+      "css": [
+        "styles.css"
+      ]
+    }
+  },
+  "styles.css": "styles.css"
+}
```

Comment:
I don't think the contents of the `public` directory should be here

Reply:
🤔

Reply:
fixed this, example was expecting a `dist` folder, webpack is outputting to `public`